gazenot/dist-manifest.json

```diff
@@ -0,0 +1 @@
+{ }
```

Comment:
Was this a mistake?

Reply:
kind-of, it's there for main.rs, which is also to-be-deleted.